.gitignore

```diff
@@ -65,6 +65,8 @@ gcovr-unit-report.xml
 junit_results.xml
 cppunit_results.xml
 boost_test_results.xml
+**/.objs
```

Comment:
The `.objs` sub-directories are supposed to be committed to the repo.  The build system doesn't create them.  I guess it could, but that would require more changes to the make rules.

The reason you saw an untracked `.objs` directory was that the whole `autoconfig` program was removed in #1320.

Reply:
Ok. thank you for the info. I'll revert the change.